geowave-analytics/src/main/java/mil/nga/giat/geowave/analytics/extract/TimeDimensionExtractor.java

```diff
@@ -0,0 +1,52 @@
+package mil.nga.giat.geowave.analytics.extract;
+
+import java.util.Calendar;
+import java.util.Date;
+
+import mil.nga.giat.geowave.store.TimeUtils;
+
+import org.opengis.feature.simple.SimpleFeature;
+import org.opengis.feature.type.AttributeDescriptor;
+
+/**
+ * 
+ * A default implementation that averages all time attributes.
+ * 
+ */
+public class TimeDimensionExtractor extends
```

Comment:
yes.  Provides an out-of-the-box option and demonstrates how the feature of multiple dimensions may be used.